src/custom-element.js

```diff
@@ -434,67 +478,103 @@ function createBaseCustomElementClass(win) {
+        })
+        .then(
+          () => {
+            this.preconnect(/* onLayout */ false);
+            this.built_ = true;
+            this.classList.add('i-amphtml-built');
+            this.classList.remove('i-amphtml-notbuilt');
+            this.classList.remove('amp-notbuilt');
+            this.signals_.signal(CommonSignals.BUILT);
+            if (this.isConnected_) {
+              this.connected_();
+            }
+            if (this.actionQueue_) {
+              // Only schedule when the queue is not empty, which should be
+              // the case 99% of the time.
+              Services.timerFor(toWin(this.ownerDocument.defaultView)).delay(
+                this.dequeueActions_.bind(this),
+                1
+              );
+            }
+            if (!this.getPlaceholder()) {
+              const placeholder = this.createPlaceholder();
+              if (placeholder) {
+                this.appendChild(placeholder);
               }
-            });
-        }
-      }).then(
-        () => {
-          this.preconnect(/* onLayout */ false);
-          this.built_ = true;
-          this.classList.add('i-amphtml-built');
-          this.classList.remove('i-amphtml-notbuilt');
-          this.classList.remove('amp-notbuilt');
-          this.signals_.signal(CommonSignals.BUILT);
-          if (this.isConnected_) {
-            this.connected_();
-          }
-          if (this.actionQueue_) {
-            // Only schedule when the queue is not empty, which should be
-            // the case 99% of the time.
-            Services.timerFor(toWin(this.ownerDocument.defaultView)).delay(
-              this.dequeueActions_.bind(this),
-              1
+            }
+          },
+          (reason) => {
+            this.signals_.rejectSignal(
+              CommonSignals.BUILT,
+              /** @type {!Error} */ (reason)
             );
-          }
-          if (!this.getPlaceholder()) {
-            const placeholder = this.createPlaceholder();
-            if (placeholder) {
-              this.appendChild(placeholder);
+            if (!isBlockedByConsent(reason)) {
+              reportError(reason, this);
             }
+            throw reason;
           }
-        },
-        (reason) => {
-          this.signals_.rejectSignal(
-            CommonSignals.BUILT,
-            /** @type {!Error} */ (reason)
-          );
-          if (!isBlockedByConsent(reason)) {
-            reportError(reason, this);
+        )
+        .then(() => {
+          if (this.isDeferredBuild()) {
+            if (deferredBuilder) {
+              deferredBuilder.unschedule(this);
+            }
+            // DO NOT SUBMIT: validate the auto-layout case.
+            this.getResources().forceLoad(this);
           }
-          throw reason;
+        });
+      return (this.buildingPromise_ = buildPromise);
+    }
+
+    /**
+     * Forces immediate load of this element.
+     * @return {!Promise}
+     */
+    load() {
```

Comment:
Why are we exposing this?

Reply:
Not sure about this yet. I've been going back and force on this. Is the "loaded" an exposed state? Then I think we should expose it. The reason I exposed it for now is because some elements have to unload to pause. E.g. an iframe, or an ad. So if it's unloaded, then "someone" should be able to load them again. I'll work more on this and we can discuss.